doc/source/examples/analysis/alignment_and_rms/pairwise_rmsd.ipynb

```diff
@@ -63,21 +69,24 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "u = mda.Universe(PSF, DCD)  # closed AdK (PDB ID: 1AKE)"
+    "adk_open = mda.Universe(CRD, DCD2)\n",
+    "adk_closed = mda.Universe(PSF, DCD)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Pairwise RMSD"
+    "## Pairwise RMSD of a trajectory to itself"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Pairwise RMSDs are an effective way to quickly view similarities and differences in conformations (as measured by RMSD) across a trajectory.\n",
+    "While 1-dimensional RMSD is a quick way to estimate how much a structure changes over time, it can be a misleading measure. It is easy to think that two structures with the same RMSD from the reference frame are also similar; but in fact, they can be very different. Instead, calculating the RMSD of each frame in the trajectory to each other frame in the reference can contain much more information. This measure is often called the pairwise, all-to-all, or 2D RMSD.\n",
```

Comment:
1) to avoid saying each twice in the sentence, maybe "each frame in the trajectory to all other frame..".
2) "frame in the reference" is a little bit confusing here since we discuss the "reference frame" above, is it necessary or can it just be left as "in the trajectory"?